css/style.css

```diff
@@ -0,0 +1,75 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+#main{
+    margin: 20px auto;
```

Comment:
add two spaces of indentation. Do not use the tab key.

Reply:
- [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.